pages/linux/urpmf.md

```diff
@@ -0,0 +1,22 @@
+# urpmf
+
+> Mageia's command for finding files in packages and querying information about them.
+> NOTE: Mageia documentation uses medium and repository as synonymous.
```

Comment:
```suggestion
```
 I don't think it is required to mention it here, as this command's page doesn't reference both of them.